src/page/sm-test/publishStreamManagerSocialPusher/index.js

```diff
@@ -293,6 +298,66 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 			console.log('Error: Failed to POST Social Pusher provision: ' + result.status)
 			alert("Error: Failed to POST Social Pusher provision" + result.status + " " + result.statusText);
 		}
+		*/
+
+		// new synchronous
+		while (attempts < attemptLimit) {
+			const xhr = new XMLHttpRequest();
+			try {
+				// Open the synchronous request
+				xhr.open('POST', url, false); // false makes it synchronous
```

Comment:
Synchronous XHR is pretty much deprecated from browsers. It probably is still supported but is recommended to use async/await `fetch` - mostly because this is UI blocking.